src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -510,4 +779,260 @@ namespace OpenLoco::ObjectManager
 
         return entry;
     }
+
+    // 0x00472D3F
+    static void markInUseObjects([[maybe_unused]] std::span<SelectedObjectsFlags> objectFlags)
```

Comment:
this method is quite large - is it possible to break into smaller pieces?

Reply:
Seconding this remark.